index.html

```diff
@@ -11,7 +11,15 @@
     />
   </head>
   <body>
-    <header></header>
+    <header>
+      <img src="assets/img/booki.png" alt="logo Booki" />
```

Comment:
Alt must be a phrase like "Logo de Booking"

Reply:
alt value modified